synthetic_data/generators.py

```diff
@@ -105,49 +107,46 @@ def _generate_uncorrelated_column_data(self, num_samples):
             col_ = copy.deepcopy(col)
 
             generator_name = col_.get("data_type", None)
-
-            if not generator_name:
-                logging.warning(
-                    f"Generator of type {generator_name} is not implemented."
-                )
-                continue
+            column_header = col_.get("column_name", None)
 
             col_["rng"] = self.rng
             col_["num_rows"] = num_samples
+            if generator_name:
```

Comment:
This is needed in place of the removed warning, as we account for a fully nulled column